### ISSUE TITLE
How to set learning rate in a big net for fine-tuning?

### ISSUE THREAD

**Issue by nannanmath** [BVLC/caffe#3509]
I have a big net with many layers. I add a new full-connection layer in the net and want to do a fine-tuning. However, it's so difficult to set lr_mult=0 in every layer except the new one, since there are many layers in the net. If there is a good way to solve these problem?

Thanks.

**Comment by ajdroid** [comment]
Try [caffe-users mailing list](https://groups.google.com/forum/#!forum/caffe-users). Close this issue as this isn't the right forum for it.

**Comment by seanbell** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
